src/IO/ADIOS/ADIOS1IOHandler.cpp

```diff
@@ -218,8 +219,8 @@ ADIOS1IOHandlerImpl::init()
 #endif
 
 #if openPMD_HAVE_ADIOS1
-ADIOS1IOHandler::ADIOS1IOHandler(std::string const& path, AccessType at)
-        : AbstractIOHandler(path, at),
+ADIOS1IOHandler::ADIOS1IOHandler(std::string path, AccessType at)
+        : AbstractIOHandler(std::move(path), at),
```

Comment:
You'd hope today's compilers were all smart enough to do this on their own, but it's good to state the intention nonetheless.